pkg/apis/pipeline/v1beta1/taskrun_validation.go

```diff
@@ -93,6 +97,38 @@ func (ts *TaskRunSpec) Validate(ctx context.Context) (errs *apis.FieldError) {
 	return errs
 }
 
+// validateInlineParameters validates parameters if embedded
```

Comment:
Same comments as for inline parameter validation-- could you please add more detail to this docstring, and see if it can be combined with any existing parameter validation?